src/main/resources/GoogleApps/GoogleAppsConfigClass.xml

```diff
@@ -367,6 +367,36 @@
           hintTextOff = "${escapetool.javascript($services.localization.render('GoogleApps.GoogleAppsConfigClass_domain.hintTextOff'))}",
           hintTextInvaliddomain = "${escapetool.javascript($services.localization.render('GoogleApps.GoogleAppsConfigClass_domain.hintTextInvaliddomain'))}";
 
+          function updateScopes() {
+            var scopes = "";
+            jQuery(("input[name^='scope_']")).each(function() {
+            if(jQuery.attr(this, "disabled")!=="disabled" &amp;&amp; this.checked)
+              scopes = scopes + " " + this.name.substring('scope_'.length);
+            });
+            jQuery("input[name$='_scope']").val(scopes);
+            updateDriveNowWhat();
+          }
+
+          function readScopes() {
+            var scope = jQuery("input[name$='_scope']").val();
+            jQuery(("input[name^='scope_']")).each(function() {
+            if(jQuery.attr(this, "disabled")!=="disabled")
+            var n = this.name.substring('scope_'.length);
```

Comment:
This line is not indented nor wrapped in `{}` so it's harder to read.

Reply:
thanks for the catch. Fixed.